pkg/virt-handler/node-labeller/cpu_plugin.go

```diff
@@ -135,7 +138,8 @@ func (n *NodeLabeller) loadHostSupportedFeatures() error {
 
 	usableFeatures := make([]string, 0)
 	for _, f := range hostFeatures.Feature {
-		if f.Policy != util.RequirePolicy {
+		// On s390x, the policy is not set
+		if f.Policy != util.RequirePolicy && !(virtconfig.IsS390X(n.arch) && f.Policy == "") {
```

Comment:
This seems to change the logic for x86

Reply:
You are right, tracking brackets might be hard sometimes, at least for me...

Reply:
Since i remember getting this particular one wrong when i first implemented it, i can only agree to that.

Reply:
> What about simply add another if statement

@alicefr unfortunately, that won't make it better. In your suggestion, the code would still skip the append, as when `f.Policy == ""` it is also `f.Policy != "require"`. Additionally i don't specifically want to disallow the `Policy=require` part on s390x, in case libvirt starts setting it in the future.

What might make sense though would be to change the patter so we only append on true, instead of skipping the loop.
So like:
```
		// On s390x, the policy is not set
		if f.Policy == util.RequirePolicy || (virtconfig.IsS390X(n.arch) && f.Policy == "") {
			usableFeatures = append(usableFeatures, f.Name)
		}
```
Or use variables instead of brackets to make it more descriptive:
```
hasPolicyRequired:=f.Policy == util.RequirePolicy
noPolicyOnS390X:=virtconfig.IsS390X(n.arch) && f.Policy == ""

if hasPolicyRequired && noPolicyOnS390X {
```
WDYT @alicefr @xpivarc

Reply:
Make it positive instead of the negative condition makes it for sure more readable.
> ```
>if hasPolicyRequired && noPolicyOnS390X {
>````

I think this should be a or

Reply:
I have switched to a positive condition. If you want i can also abstract it/make it more descriptive via variables or make my comment more verbose.

> I think this should be a or

Yes it should.

Reply:
that's fine now, it is more readable, thanks